packages/web/app/pages/[orgId]/view/subscription/manage.tsx

```diff
@@ -202,14 +193,7 @@ const Inner = ({
     updateOrgRateLimitMutation[0].error;
 
   return (
-    <DataWrapper
-      query={query}
-      loading={
-        upgradeToProMutation[0].fetching ||
-        downgradeToHobbyMutation[0].fetching ||
-        updateOrgRateLimitMutation[0].fetching
-      }
-    >
+    <DataWrapper query={query}>
```

Comment:
we had 2 same DataWrapper's and DataWrapper from v1 contained a `loading` prop, but in DataWrapper of v2 there is no this prop, so I just removed it